host/create.c

```diff
@@ -1416,6 +1416,8 @@ oe_result_t oe_create_enclave(
     uint32_t flags,
     const void* config,
     uint32_t config_size,
+    void* ocall_table,
```

Comment:
why is ocall_table not of type oe_ocall_func_t*?

Reply:
Good point. I have made it strongly-types. I have used the type inline:
void  (**ocall_table(void*)
instead of including edger8r/host.h to keep a strong separation between what a programmer would use (host.h) and what edger8r would use (edger8r/host.h).